### ISSUE TITLE
No train code

### ISSUE THREAD

**Issue by xup16** [MarcoForte/FBA_Matting#1]
Hi~ Thank you for the wonderful work.
I find your work on the alphamatting.com, and I am very glad that you open source your code so quickly. 
However, I do not find the training code in the repository. 
Would you like also open source the training code?

**Comment by MarcoForte** [comment]
Hi unfortunately I cannot release the training code at this time. I've updated the readme with some key tips. I encourage people to try apply these and our network architecture to existing open source training codes. For example,
- https://github.com/Yaoyi-Li/GCA-Matting
- https://github.com/foamliu/Deep-Image-Matting-PyTorch
- https://github.com/huochaitiantang/pytorch-deep-image-matting

**Comment by ucb-pb** [comment]
Thanks for everything you have provided. Very impressive. What do you mean by clipping the alpha? What do you do?

**Comment by MarcoForte** [comment]
torch.clamp(alpha,0,1)

**Comment by ucb-pb** [comment]
Ok, I'm unclear about where that would happen. Like is it applied to the output alpha before the loss is calculated?

**Comment by MarcoForte** [comment]
Hi yes it's applied to the output of the network for the alpha channel
before the loss is calculated.

Occasionally this can cause issues at the beginning of training. And the
output defaults to either all zeroes or all ones. If this happens, disable
the clamping for the first few hundred iterations and then switch it back
on.

**Comment by ucb-pb** [comment]
thank you. I see the clamping and group normalization are already incorporated into your network code, which is great. is the weight standardization happening too? I can't tell where that would/should happen at first glance if it is.

**Comment by ucb-pb** [comment]
actually, i see clamping in a few places. which one(s) do you disable?
